src/shared/ui/survey/SimpleTextInput.tsx

```diff
@@ -0,0 +1,55 @@
+import { FC, useState } from 'react';
+import { TextInputProps } from 'react-native';
+
+import { useTranslation } from 'react-i18next';
+
+import { colors } from '@shared/lib';
+import { Input } from '@shared/ui';
+
+type ValueType = '' | 'http://www.w3.org/2001/XMLSchema#integer';
```

Comment:
I may misinterpret the need for it but I think we can use the following type:
`type KeyboardType = 'text' | 'numeric'`;

Reply:
We get exactly "valueType" prop from backend, and then I "convert" it to keyboardType.

Reply:
Then backend should be changed

Reply:
Agreed. @vmkhitaryanscn,  please contact @Tom1204 in order to change the data contract